providers/hubspot.json

```diff
@@ -13,6 +13,11 @@
       "type": "apiKey",
       "in": "query",
       "name": "hapikey"
+    },
+    {
```

Comment:
note: Keep in mind that provider will not be updated in Station CI/CD pipeline. Update can be done using backoffice API, I can help with that.

Reply:
I know :(

Thanks!